### ISSUE TITLE
Where is CONFIG file for Airflow?

### ISSUE THREAD

**Issue by UmutAlihan** [anilkulkarni87/airflow-docker#1]
I can't find the config file for Airflow deployment...

Where is it ?

**Comment by anilkulkarni87** [comment]
@UmutAlihan  Can you provide an example of what config you would want to change?

**Comment by anilkulkarni87** [comment]
@UmutAlihan  I am not using the config file. some of the configs are part of the docker-compose yaml file.

**Comment by UmutAlihan** [comment]
yes I realized that it is mostly confugured via env vars on compose file, much aooreciated thabk you
